pyproject.toml

```diff
@@ -18,6 +18,7 @@ exclude = '''
     | docs
     | .enterprise
     | enterprise/src
+    | enterprise/version.py
```

Comment:
I have no idea why but this should look like:
```python
exclude = '''
(
  /(
      \.eggs         # exclude a few common directories in the
    | \.git          # root of the project
    | \.hg
    | \.mypy_cache
    | \.tox
    | _build
    | buck-out
    | build
    | dist
    | docs
    | .enterprise
    | enterprise/src
  )/
  | enterprise/version.py
)
'''
```
This works. We probably also want to change the line in the CI that installs black to just do `python -m pip install -r requirements_dev.txt` since that uses a locked down version of black.

Reply:
Indeed it works in my fork, thanks! I will close this PR, which has become a conflation of three different development efforts in my fork, and I will try to make separate PRs.